app/src/main/res/drawable/ic_close_small.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Run `avocado` on this `VectorDrawable` 🙂 https://github.com/alexjlockwood/avocado